ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKGroupedStreamTest.java

```diff
@@ -107,6 +108,7 @@
 
   @Before
   public void setUp() {
+    when(keyFormat.getFormat()).thenReturn(FormatFactory.KAFKA);
```

Comment:
Any reason we don't remove `getFormat` and inline using `getFormatInfo` here, similar to the changes we recently made to the source properties classes?

Reply:
That's coming in the next PR :)